package.json

```diff
@@ -102,12 +102,19 @@
       {
         "permissionName": "module.eholdings.enabled",
         "displayName": "UI: eHoldings module is enabled",
-        "visible": true,
+        "visible": false,
         "subPermissions": [
           "kb-ebsco.all",
           "tags.all"
         ]
       },
+      {
+        "permissionName": "ui-eholdings.app.enabled",
+        "subPermissions": [
+          "module.eholdings.enabled"
+        ],
+        "visible": true
+      },
       {
         "permissionName": "settings.eholdings.enabled",
         "displayName": "Settings (eHoldings): display list of settings pages",
```

Comment:
Shouldn't this permission be hidden too since now we have `ui-eholdings.settings.enabled` which is meant to be shown in the permissions list?

Reply:
By the default `visible` gets false, but you are right, it will better to set this param, to make it more readable and not confuse other developers

Reply:
My bad. It seemed to me that ‘visible: true’ was written there